packages/ui/src/components/Menu/index.tsx

```diff
@@ -1,43 +1,36 @@
-import { Menu as HeadlessMenu } from '@headlessui/react';
+import * as RadixMenu from '@radix-ui/react-dropdown-menu';
 import { useRouter } from 'next/router';
 import { MouseEventHandler, ReactElement, ReactNode } from 'react';
 
 type MenuProps = {
   children: ReactNode;
 };
 
-export const Menu = ({ children, ...props }: MenuProps) => {
+export const Menu = ({ children }: MenuProps) => {
   return (
-    <HeadlessMenu as="div" className="relative" {...props}>
```

Comment:
I don't understand why there's a `relative` property if we're not using `absolute` anywhere.

Reply:
Also, what does the `div` do?

Reply:
Without the `as="div"`, headless would not wrap the children:

![image](https://user-images.githubusercontent.com/43268759/205510531-29b56b31-4c84-4cd2-9eb8-7d3ed9d579d8.png)

I think we can safely remove this `relative` class